src/pages/debug-app/debug-app.tsx

```diff
@@ -426,7 +439,7 @@ export const DebugApp = () => {
   };
 
   const renderTilePanel = () => {
-    if (!selectedTile) {
+    if (!selectedTile?.selected) {
```

Comment:
Don't need this check anymore. If the tile in `selectedTiles` array, it must be `selected`